src/stars.cpp

```diff
@@ -509,7 +509,7 @@ void CPL_write_gdal(NumericMatrix x, CharacterVector fname, CharacterVector driv
 		eType = GDT_Byte; // #nocov
 #if GDAL_VERSION_NUM >= 3070000
 	else if (Type[0] == "Int8")
-		eType = GDT_Int8; // #nocov
+		eType = GDT_Byte; // #nocov
```

Comment:
This doesn't look good IMO, see https://github.com/r-spatial/sf/issues/2033 - do you actually run into this code? That would mean the #if condition is wrong.

Reply:
GDT_Int8 implementation has now landed in GDAL master